functions/src/export-csv.ts

```diff
@@ -86,23 +87,46 @@ export async function exportCsvHandler(
   const csvStream = csv.format({
     delimiter: ',',
     headers,
-    includeEndRowDelimiter: true,
     rowDelimiter: '\n',
-    quoteColumns: true,
-    quote: '"',
+    includeEndRowDelimiter: true, // Add \n to last row in CSV
+    quote: false,
   });
   csvStream.pipe(res);
 
   const submissionsByLoi = await getSubmissionsByLoi(survey.id, jobId);
 
   loiDocs.forEach(loiDoc => {
-    submissionsByLoi[loiDoc.id]?.forEach(submission =>
-      writeRow(csvStream, loiProperties, tasks, loiDoc, submission)
+    const loi = toMessage(loiDoc.data(), Pb.LocationOfInterest);
+    if (loi instanceof Error) {
+      throw loi;
+    }
+    const submissions = submissionsByLoi[loiDoc.id] || [{}];
```

Comment:
Why not default to empty list and skip the `forEach`?

Reply:
This effectively ensures we include one row for jobs without submissions (outer join on jobs). Definitely deserves a comment - adding one now!